server/plugin/plugin.go

```diff
@@ -323,11 +326,11 @@ func (p *Plugin) initEnv(e *Env, pluginURL string) error {
 		e.bot = bot.New(p.API, pluginURL)
 		err := e.bot.Ensure(
 			&model.Bot{
-				Username:    config.BotUserName,
-				DisplayName: config.BotDisplayName,
-				Description: config.BotDescription,
+				Username:    e.Provider.BotUsername,
+				DisplayName: e.Provider.BotDisplayName,
+				Description: fmt.Sprintf(config.BotDescription, e.Provider.DisplayName),
 			},
-			filepath.Join("assets", "profile.png"),
+			filepath.Join("assets", fmt.Sprintf("profile-%s.png", e.Provider.DisplayName)),
```

Comment:
I think we want `Name` here for the icon, i.e. `gcal` and `mscalendar`

```suggestion
			filepath.Join("assets", fmt.Sprintf("profile-%s.png", e.Provider.Name)),
```